sphinx_gherkindoc/parsers/pytest_bdd.py

```diff
@@ -133,9 +136,22 @@ class Feature(PytestModel):
     """Feature model for Pytest-Bdd."""
 
     def __init__(self, root_path: str, source_path: str):
-        self._data = pytest_bdd.feature.Feature(
-            root_path, pathlib.Path(source_path).resolve().relative_to(root_path)
-        )
+        relative_file_path = pathlib.Path(source_path).resolve().relative_to(root_path)
+
+        # If the version of pytest-bdd implements the parse_feature method then use it
+        pytest_bdd_parse_feature = getattr(pytest_bdd.feature, "parse_feature", None)
```

Comment:
I wonder if we want to/can change https://github.com/jolly-good-toolbelt/sphinx_gherkindoc/blob/master/pyproject.toml#L25 so that this tool removes support for the older pytest-bdd versions in its next published release.

If it is a hassle (I think you have a corresponding pytest-bdd PR open?), I think supporting both for now is fine.